src/python/pants/option/global_options.py

```diff
@@ -769,10 +825,32 @@ def register_bootstrap_options(cls, register):
                 "Path to a file containing an oauth token to use for gGRPC connections to "
                 "--remote-execution-server and --remote-store-server.\n\nIf specified, Pants will "
                 "add a header in the format `authorization: Bearer <token>`. You can also manually "
-                "add this header via `--remote-execution-headers` and `--remote-store-headers`. "
+                "add this header via `--remote-execution-headers` and `--remote-store-headers`, or "
+                "use `--remote-auth-plugin` to dynamically set the relevant headers. "
```

Comment:
Maybe change to  "use `--remote-auth-plugin` to provide a plugin to dynamically set the relevant headers. " ?